lib/android/app/src/main/java/com/reactnativenavigation/views/Container.java

```diff
@@ -0,0 +1,7 @@
+package com.reactnativenavigation.views;
+
+import com.reactnativenavigation.viewcontrollers.ContainerViewController;
+
+public interface Container extends ContainerViewController.IReactView {
+    TopBar getTopBar();
```

Comment:
add TODO with description to refactor this breakage of SRP

Reply:
or open issue

Reply:
Good point. #2348